commons/src/main/scala/com/lynbrookrobotics/potassium/commons/electronics/DynamicBraking.scala

```diff
@@ -0,0 +1,47 @@
+package com.lynbrookrobotics.potassium.commons.electronics
+
+import squants.QuantityRange
+import squants.electro.{ElectricPotential, Volts}
+import squants.motion.AngularVelocity
+
+object DynamicBraking {
+  private val ditherPatterns = List(
+    "............", // 0
+    "|...........", // 1
+    "|.....|.....", // 2
+    "|...|...|...", // 3
+    "|..|..|..|..", // 4
+    "|.|.|.|..|..", // 5
+    "|.|.|.|.|.|.", // 6
+    "||.||.|.|.|.", // 7
+    "||.||.||.||.", // 8
+    "|||.|||.|||.", // 9
+    "|||||.|||||.", // 10
+    "|||||||||||.", // 11
```

Comment:
Can we review this in person? This seems like a strange thing to do

Reply:
+1 This could probably just be represented by a `Byte` like in the original implementation https://github.com/Team846/code-2014/blob/7f0c8ee1bb4b1678708b78b101f46959afdb1b41/LRT14/Actuators/DriveESC.cpp#L107

Reply:
I was talking to Mr. G about this:
- A longer pattern length allows for more intermediate brake strengths, therefore smoother deceleration.
- Writing the patterns in hex is unnecessary. It requires documentation and is hard to visualize.  

Because [Scala doesn't have binary literals](https://stackoverflow.com/questions/7197119/how-to-write-binary-literals-in-scala), I decided to use a String to store braking patterns.

Reply:
The longer pattern makes sense. Now that I think of it, we should really put together a proper algorithm for calculating the brake patterns. The patterns themselves are pretty systematic so there's no reason they need to be calculated by humans.